spec/factories/book.rb

```diff
@@ -3,7 +3,7 @@
     genre { Faker::Book.genre }
     author { Faker::Book.author }
     image { Faker::File.file_name(dir: 'path/to') }
-    title { Faker::Book.title }
+    title { Faker::Books::Lovecraft.tome }
```

Comment:
:thinking: this faker always be lower than 25? 

```suggestion
    title { Faker::Books::Lovecraft.tome.first 25 }
```